Sources/MountebankSwift/Models/Recreatable.swift

```diff
@@ -0,0 +1,176 @@
+    }
+
+    func classSwiftString(depth: Int, _ properties: [(String?, Recreatable?)]) -> String {
+        structOrClassSwiftString(depth: depth, properties: properties)
+    }
+
+    private func structOrClassSwiftString(depth: Int, properties: [(String?, Recreatable?)]) -> String {
+        let newDepth = depth + 1
+        return """
+        \(Self.self)(\n\(Self.propertyListSwiftString(depth: newDepth, properties: properties))
+        \(String(repeating: INDENTATION, count: depth)))
+        """
+    }
+
+    func enumSwiftString(depth: Int, _ properties: [(Recreatable?)] = []) -> String {
+        enumSwiftString(depth: depth, properties.map { (nil, $0) })
+    }
+
+    func enumSwiftString(depth: Int, _ properties: [(String?, Recreatable?)]) -> String {
+        let enumCaseWithoutAssociatedValues = "\(self)".split(separator: "(").first.map(String.init) ?? "\(self)"
+
+        if properties.isEmpty {
+            return ".\(enumCaseWithoutAssociatedValues)"
+        }
+
+        let newDepth = depth + 1
+        let propertiesSwiftString = Self.propertyListSwiftString(depth: newDepth, properties: properties)
+        if propertiesSwiftString.isEmpty {
+            return ".\(enumCaseWithoutAssociatedValues)()"
+        } else {
+            return """
+            .\(enumCaseWithoutAssociatedValues)(\n\(propertiesSwiftString)
+            \(String(repeating: INDENTATION, count: depth)))
+            """
+        }
+    }
+
+    fileprivate static func propertyListSwiftString(depth: Int, properties: [(String?, Recreatable?)]) -> String {
+        properties.map { propertySwiftString(depth: depth, $0, $1) }
+            .filter{ !$0.isEmpty }
+            .joined(separator: DELIMITER)
+    }
+
+    fileprivate static func propertySwiftString(depth: Int, _ label: String?, _ value: Recreatable?) -> String {
+        guard let value else {
+            return ""
+        }
+
+        let swiftString = value.swiftString(depth: depth)
+        guard !swiftString.isEmpty else {
+            return ""
+        }
+
+        if let label {
+            return "\(String(repeating: INDENTATION, count: depth))\(label): \(swiftString)"
+        } else {
+            return "\(String(repeating: INDENTATION, count: depth))\(swiftString)"
+        }
+    }
+}
+
+// MARK: - Extensions
+
+extension String: Recreatable {
+    public func swiftString(depth: Int) -> String {
+        debugDescription
+    }
+}
+
+extension Int: Recreatable {
+    public func swiftString(depth: Int) -> String {
+        description
+    }
+}
+
+extension Double: Recreatable {
+    public func swiftString(depth: Int) -> String {
+        debugDescription
+    }
+}
+
+extension Bool: Recreatable {
+    public func swiftString(depth: Int) -> String {
+        description
+    }
+}
+
+extension Data: Recreatable {
+    public func swiftString(depth: Int) -> String {
+        "Data(base64Encoded: \(base64EncodedString().debugDescription))!)"
```

Comment:
Data is kind of not a nice format to display as a string.. this is the best I could think of.